### ISSUE TITLE
Colour picker - Dragging RGB channels doesn't drag the value down to 0

### ISSUE THREAD

**Issue by DarrenVs** [WootingKb/wootility-issues#42]
**Describe the bug**
Dragging the RGB channels in the colour picker doesn't allow the value to become 0.

**To Reproduce**
1. Hover your mouse over the R, G, or B letter in the colour picker until your pointer icon changes to a drag icon
2. Drag your mouse all the way to the left trying to make the value `0`
3. Notice that you can only get it down to `1`

**Video**
![5pnbSsloAg](https://user-images.githubusercontent.com/2780639/85277195-48019480-b483-11ea-814a-5e1bb5e8ab58.gif)

**Workaround**
- Clicking on the number and inserting `0` as your value

---

**Desktop**
 - OS: Windows 10

**Wootility and Firmware versions**

- Frimware: `1.29.0`
- Wootility: `3.5.7-alpha`

**Keyboard Type**

- Wooting One

**Comment by simon-wh** [comment]
From my investigation this is completely down to the [library we're using for the color picker](https://github.com/casesandberg/react-color). You can even see that behaviour on the demo on their website. 

I also noticed that the text you drag on is always black & doesn't change based on theme. Turns out, the library has no mechanism for applying custom styling to labels :angry:

**Comment by PastaJ36** [comment]
Yeah this library is not good. We already have some pretty hacky CSS to get the current styling. We choose that one all the way at the very start, there has to be a better one by now
